tpl/bootstrap.php

```diff
@@ -123,46 +123,52 @@
 				</div>
 			</div>
 		</div>
+		<div id="loadconfirmmodal" tabindex="-1" class="modal fade" role="dialog" aria-hidden="true">
+			<div class="modal-dialog" role="document">
+				<div class="modal-content">
+					<div class="modal-header">
+						<button type="button" class="close" data-dismiss="modal" aria-label="<?php echo I18n::_('Close') ?>"><span aria-hidden="true">&times;</span></button>
+						<h4 class="modal-title"><?php echo I18n::_('Burn after reading pastes can only be displayed once upon loading it. Do you want to open it now?') ?></h4>
+					</div>
+					<div class="modal-body text-center">
+						<button id="loadconfirm-open-now" type="button" class="btn btn-success" data-dismiss="modal"><span class="glyphicon glyphicon-download"></span> <?php echo I18n::_('Yes, load it') ?></button>
+					</div>
+				</div>
+			</div>
+		</div>
 <?php
 if ($QRCODE) :
 ?>
-		<div id="qrcodemodal" tabindex="-1" class="modal fade" aria-labelledby="qrcodemodalTitle" role="dialog" aria-hidden="true">
```

Comment:
Seems you removed `aria-labelledby`. Was this intentional? And does not this negatively affect accessibility?

Reply:
It was - it only makes sense to have that if it does reference an ID that contains an area label. Those IDs do not exist, so it didn't actually add any value to visually impaired readers. I did add actual area labels to the new "Close" buttons and a proper title to the QR code modal. Before this change, the modal would open and only read "Close" for the close button, but the inserted QR code doesn't provide any readable content. Now at least it will read "QR code - pause - Close".